python/pyspark/ml/feature.py

```diff
@@ -926,125 +1072,135 @@ class CountVectorizer(JavaEstimator, _CountVectorizerParams, JavaMLReadable, Jav
     def __init__(
         self,
         *,
-        minTF=1.0,
-        minDF=1.0,
-        maxDF=2 ** 63 - 1,
-        vocabSize=1 << 18,
-        binary=False,
-        inputCol=None,
-        outputCol=None,
-    ):
+        minTF: float = 1.0,
+        minDF: float = 1.0,
+        maxDF: float = 2 ** 63 - 1,
+        vocabSize: int = 1 << 18,
+        binary: bool = False,
+        inputCol: Optional[str] = None,
+        outputCol: Optional[str] = None,
+    ) -> None:
         """
         __init__(self, \\*, minTF=1.0, minDF=1.0, maxDF=2 ** 63 - 1, vocabSize=1 << 18,\
                  binary=False, inputCol=None,outputCol=None)
         """
         super(CountVectorizer, self).__init__()
         self._java_obj = self._new_java_obj("org.apache.spark.ml.feature.CountVectorizer", self.uid)
-        kwargs = self._input_kwargs
+        kwargs = self._input_kwargs  # type: ignore[attr-defined]
```

Comment:
`_input_kwargs` annotations should be added explicitly for classes that use them. You already have them for some of the definitions in this module.